### PR TITLE
Make airflow dags show display TaskGroup.

### LINKED ISSUE #13053: CLI does not display TaskGroups
Hello,

Airflow ability to [display DAG in CLI](http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/usage-cli.html#display-dags-structure) with command `airflow dags show`, but unfortunately this command does not display Task Groups. It would be great if the Task Groups were correctly marked in the diagrams.

Screenshot 2020-12-14 at 02 28 58
Screenshot 2020-12-14 at 02 29 16

Best regards,
Kamil Breguła

### PR DESCRIPTION
**Pull request by yuqian90**
closes: #13053

Make `airflow dags show` display `TaskGroup`.

Example output:
image

### CODE REVIEW

**Comment by yuqian90** [comment]
@mik-laj @kaxil could you take a look?

File: airflow/utils/dot_renderer.py [hunk 1]
```diff
@@ -42,6 +45,80 @@ def _refine_color(color: str):
...
+        parent_graph.node(
+            node.task_id,
+            _attributes={
+                "label": node.label,
+                "shape": "rectangle",
+                "style": "filled,rounded",
+                "color": _refine_color(color),
+                "fillcolor": _refine_color(fill_color),
+            },
+        )
```
Comment (turbaszek) [inline airflow/utils/dot_renderer.py]: How about introducing `_draw_task` and `_draw_task_group` to encapsulate the logic and split logic of `_draw_nodes` into smaller pieces?

Comment (yuqian90) [reply airflow/utils/dot_renderer.py]: Thanks @turbaszek . Done

**Review by turbaszek — APPROVED** [review]

**Review by kaxil — APPROVED** [review]
🎉

**Comment by yuqian90** [comment]
Hi @Kaxil, thanks for merging this PR. Can it go into 2.0.2?

**Pull request merged**